static/js/code-highlighter.js

```diff
@@ -16,7 +16,9 @@ var Sticky = new (class Sticky {
 
     // Our logic can't work on our diff output because there will be line
     // number discontinuities and line numbers that are simply missing.
-    if (!document.getElementById("content").classList.contains("diff")) {
+    let hasLineNumbers = document.querySelector(".line-number") != null;
```

Comment:
I'd just `!!document.querySelector(".line-number")`, but this also works for me.

Reply:
Updated